addons/hr_recruitment/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,113 @@
+---Fields in module 'hr_recruitment'---
+hr_recruitment / hr.applicant             / attachment_ids (one2many)     : NEW relation: ir.attachment
+# Nothing to do
+
+hr_recruitment / hr.applicant             / availability (integer)        : type is now 'date' ('integer')
+# Unable to calculate date form a integer --> Nothing to do
```

Comment:
You should take field `create_date` and add this number of days to get the new date.